runtime/shared_common/sharedconsts.h

```diff
@@ -92,6 +92,7 @@
 #define PRINTSTATS_SHOW_ALL_STALE 0x40000
 #define PRINTSTATS_SHOW_STARTUPHINT 0x80000
 #define PRINTSTATS_SHOW_TOP_LAYER_ONLY 0x100000
+#define PRINTSTATS_SHOW_CRVSNIPPET 0x200000
```

Comment:
It would be more consistent to move this up and use 0x200.
PRINTSTATS_SHOW_ALL needs to be modified.

Reply:
Will update PRINTSTATS_SHOW_ALL to include CRVSNIPPET. Looks like it's missing STARTUPHINT as well. I can add the STARTUPHINT fix in a separate PR, or just include it in my fix here?

For the const value, 0x200 to 0x10000 are already being used: https://github.com/eclipse/openj9/blob/bf20590b2547b25e0cda1eaa950a6da763101309/runtime/shared_common/sharedconsts.h#L97-L108

Reply:
Better to add STARTUPHINT in a separate PR.

Reply:
opened here: https://github.com/eclipse/openj9/pull/8439